tests/fuzz/seq_prod_fuzz_example/Makefile

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
Is this Makefile run in one of the CI tests ?

Reply:
I don't think the benefit of having this in CI is worth the cost of adding another job. Happy to discuss offline.